### PR TITLE
Add multi-zone support to create infra aws

### PR DESCRIPTION
**Pull request by sjenning**
**What this PR does / why we need it**:
Enable multi-zone infra support for guest cluster so we can start testing multizone guest clusters in CI.

Note: This does not enable the functionality e2e.  Just the infra creation.

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

**Comment by sjenning** [comment]
e2e currently assumes that NodePool.name == HostedCluster.name.  With the zone suffix on the NodePool, this is no longer true. 
/hold

File: api/fixtures/example.go [hunk 1]
```diff
@@ -308,16 +312,26 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 		},
 	}
 
-	var nodePool *hyperv1.NodePool
-	if o.NodePoolReplicas > -1 {
-		nodePool = &hyperv1.NodePool{
+	if o.NodePoolReplicas <= -1 {
```
Comment (alvaroaleman) [inline api/fixtures/example.go:315]: Did you mean < 1?

Comment (sjenning) [reply api/fixtures/example.go:315]: Flag description is `If >-1, create a default NodePool with this many replicas`

So I am inverting the logic here so I can exit early if the user doesn't what the NodePools created.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**